README.rst

```diff
@@ -41,11 +41,11 @@ The ``-w`` flag will actually implement the changes recommended by codespell. No
 
     codespell -I FILE, --ignore-words=FILE
 
-The ``-I`` flag can be used to whitelist certain words that are in the ``codespell_lib/data/dictionary.txt``. The format of the whitelist file is one word per line. Invoke using: ``codespell -I path/to/file.txt`` to execute codespell referencing said whitelist. **Important note:** The whitelist passed to ``-I`` is case-sensitive based on how it is listed in ``dictionary.txt``. ::
+The ``-I`` flag can be used to allowlist certain words that are in the ``codespell_lib/data/dictionary.txt``. The format of the allowlist file is one word per line. Invoke using: ``codespell -I path/to/file.txt`` to execute codespell referencing said allowlist. **Important note:** The allowlist passed to ``-I`` is case-sensitive based on how it is listed in ``dictionary.txt``. ::
```

Comment:
Personally I'd be tempted to say if we're rewording these why don't we drop the made up word entirely, it's not a variable so we can use a phrase that's more easily understood by translation and non-native speakers etc.

Perhaps something like:
```suggestion
The ``-I`` flag can be used for a list of certain words to allow that are in the codespell dictionaries. The format of the file is one word per line. Invoke using: ``codespell -I path/to/file.txt`` to execute codespell referencing said list of allowed words. **Important note:** The list passed to ``-I`` is case-sensitive based on how it is listed in the codespell dictionaries. ::
```